wg-ecosystem/README.md

```diff
@@ -6,19 +6,19 @@ Oversees the projects that make Electron app development easier.
 
 | Avatar | Name | Role | Time Zone |
 | -------------------------------------------|----------------------|----------------------------| -------- |
+| <img src="https://github.com/blackhole1.png" width=100 alt="@blackhole1">  | Black-Hole [@blackhole1](https://github.com/blackhole1) | Member | BJT (Hangzhou) |
```

Comment:
Why are we changing the order here? It was order-joined originally which was kinda nice

Reply:
It was already alphabetized by GitHub username for everyone else but our most recent members added themselves at the bottom so I kind of fixed that.

Reply:
That's true, considering I was second in the list. 😅